presto-jdbc/src/main/java/io/prestosql/jdbc/AbstractPrestoResultSet.java

```diff
@@ -279,7 +279,7 @@ private Date getDate(int columnIndex, DateTimeZone localTimeZone)
             return new Date(calendar.getTimeInMillis());
         }
         catch (IllegalArgumentException e) {
-            throw new SQLException("Invalid date from server: " + value, e);
+            throw new SQLException("Expected value to be a date but is : " + value);
```

Comment:
There is redundant space before the colon:

```suggestion
            throw new SQLException("Expected value to be a date but is: " + value);
```